src/core/Akka.Remote/RemoteSystemDaemon.cs

```diff
@@ -304,45 +304,26 @@ private void HandleDaemonMsgCreate(DaemonMsgCreate message)
         /// <returns>ActorRef.</returns>
         public override IActorRef GetChild(IEnumerable<string> name)
         {
-            var elements = name.ToArray();
-            var path = elements.Join("/");
```

Comment:
No need for redundant .ToArray + Join call.  All of the objects passed into this method are `IReadOnlyList<string>`